### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by NRABrasil**
to include Bronchoscopy Simulation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,6 +138,18 @@ <h2>Demos</h2>
 				</div>
 
 				<div class="gallery">
+					<article class="gallery-item 3rd" itemscope="" itemtype="http://schema.org/CreativeWork">
+						<a title="Brilliant man" class="gallery-item-link" href="http://iprojects.space/bronchoscopy/index.html" target="blank" itemprop="url">
```
Comment (deltakosh) [inline index.html]: title seems wrong :)

File: index.html [hunk 2]
```diff
@@ -138,6 +138,18 @@ <h2>Demos</h2>
...
+							<img class="gallery-item-img" alt="Bronchoscopy Simulation" src="http://iprojects.space/bronchoscopy/bjs_img.jpg" itemprop="image">
```
Comment (deltakosh) [inline index.html]: we need the image to be saved in /screenshots folder for perf reasons

**Comment by NRABrasil** [comment]
Oh yes, I will try to fix it!

Saudações

*Nelson Rodrigues Alves*
*Multimedia/Game Designer*
*+55(31)9999 2979*

* iProjects *
www.iprojects.com.br





Em sex, 22 de fev de 2019 às 19:43, David Catuhe <notifications@github.com>
escreveu:

File: index.html (hunk 2 above)
Comment (NRABrasil) [reply index.html]: bjs_002

Comment (NRABrasil) [reply index.html]: How I can upload to /screenshots ?

**Comment by deltakosh** [comment]
I'll upload it no worries

**Pull request merged**